aiogram/dispatcher/filters/builtin.py

```diff
@@ -111,6 +111,7 @@ async def check_command(cls, message: types.Message, commands, prefixes, ignore_
             return False
 
         full_command = text.split()[0]
+        args = text.split()[1:]
```

Comment:
Command arguments should not be a list because arguments can be parsed in different ways.
In most of cases it should be string (not list of strings) and this variant is most versatile.

So that's mean this code should be changed to something like this:
```python3
full_command, *args_list = text.split(maxsplit=1)
args = args_list[0] if args_list else None
```